banzai/utils/date_utils.py

```diff
@@ -82,6 +82,8 @@ def parse_epoch_string(epoch_string):
 
 
 def parse_date_obs(date_obs_string):
+    if date_obs_string == 'N/A':
+        return None
```

Comment:
Maybe log a warning in this case?

Reply:
I have mixed feelings on this. I think not because all frames that are taken manually will produce a warning. I think if there is a problem, the exception will be self explanatory. If we don't need it, we don't need to clutter the logs.